relay-replays/tests/fixtures/rrweb-event-5.json

```diff
@@ -1,20 +1,108 @@
-{
-    "type": 5,
-    "timestamp": 1658770772.902,
-    "data": {
-        "tag": "performanceSpan",
-        "payload": {
-            "op": "memory",
-            "description": "memory",
-            "startTimestamp": 1658770772.902,
-            "endTimestamp": 1658770772.902,
-            "data": {
-                "memory": {
-                    "jsHeapSizeLimit": 4294705152,
-                    "totalJSHeapSize": 10204109,
-                    "usedJSHeapSize": 9131621
+[
+    {
+        "type": 5,
+        "timestamp": 1658770772.902,
+        "data": {
+            "tag": "performanceSpan",
+            "payload": {
+                "op": "memory",
+                "description": "memory",
+                "startTimestamp": 1658770772.902,
+                "endTimestamp": 1658770772.902,
+                "data": {
+                    "memory": {
+                        "jsHeapSizeLimit": 4294705152,
+                        "totalJSHeapSize": 10204109,
+                        "usedJSHeapSize": 9131621
+                    }
+                }
+            }
+        }
+    },
+    {
+        "type": 5,
+        "timestamp": 1658770772,
```

Comment:
Did this actually fail to parse as `f64`?

Reply:
@jjbayer You're right.  I had test coverage that was failing but `1658770772.0` (instead of `1658770772`) should be reasonably parse-able by the UI.  I think its okay to leave this as f64.

Reply:
`assert-json-diff` seems to have a more lenient compare mode, with which you might be able to keep the test cases:

https://docs.rs/assert-json-diff/latest/assert_json_diff/enum.NumericMode.html#variant.AssumeFloat